tools/shared.py

```diff
@@ -2587,18 +2584,17 @@ def is_ar(filename):
   @staticmethod
   def is_bitcode(filename):
     # look for magic signature
-    b = bytearray(open(filename, 'rb').read(4))
-    if len(b) < 4:
```

Comment:
looks like the length check was removed here - I suppose it's not necessary (although we read 4 chars and check the values of 2, so technically this can behave differently than before)?